src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -508,6 +506,10 @@ class ADIOSWriter : public ILightweightPlugin
             mpi_pos = gc.getPosition();
             mpi_size = gc.getGpuNodes();
 
+            /* if number of aggregators is not set we use all mpi process as aggregator*/
+            if( mThreadParams.adiosAggregators == 0 )
+               mThreadParams.adiosAggregators=mpi_size.productOfComponents();
```

Comment:
minor: spaces around `=`

general question: usually, a good and number for the aggregators, recommended by the adios devs, is `2` to `3` times the `adiosOST`'s. Therefore, I would recommend using that as an default (`mThreadParams.adiosOST * 3`).